musonant/questioner
# Ft write extra tests 162882721

#### What does this PR do?
- Update tests to be meet specification
#### Description of Task to be completed?
- Edit tests for meetups route
- Edit tests for questions route
#### How should this be manually tested?
- After cloning repository, run `npm install` to install dependencies
- Now run `npm run test` to see all test results
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162882721
#### Screenshots (if appropriate)
#### Questions:`
